include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -135,7 +135,8 @@ SECTIONS
 
 	_image_rom_start = ROM_ADDR;
 
-    SECTION_PROLOGUE(_TEXT_SECTION_NAME,,)
+    /* The exception vector table is located at the beginning of ROM */
+    SECTION_PROLOGUE(vector_table,,)
```

Comment:
riscv has the same scheme, but calls the section "vector", maybe align on that name?

Reply:
I think the name is good; it is actually the ARM Cortex-M exception vector table; maybe we could rename the Risc-V Section instead, but that's another story.